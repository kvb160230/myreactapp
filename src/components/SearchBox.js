import React from 'react';

const SearchBox = ({searchfield, searchChange}) => {
    return (
        <div className = 'pa1'>
             <input 
                className = 'pa3 ba bg-washed-blue'
                type = 'search' 
                placeholder ='search name' 
                onChange = {searchChange}
             />
        </div>
    
    );
}

export default SearchBox;