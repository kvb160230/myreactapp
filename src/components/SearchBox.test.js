import React from 'react';
import { shallow } from 'enzyme';
import SearchBox from './SearchBox';

it('expect to render Card component', () => {
  const cardComponent = shallow(<SearchBox />);
  expect(cardComponent.debug()).toMatchSnapshot();
});