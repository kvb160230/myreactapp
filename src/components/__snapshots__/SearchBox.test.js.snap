// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Card component 1`] = `
"<div className=\\"pa1\\">
  <input className=\\"pa3 ba bg-washed-blue\\" type=\\"search\\" placeholder=\\"search name\\" onChange={[undefined]} />
</div>"
`;
